katalog/elasticsearch-triple/triple.yml

```diff
@@ -9,5 +13,66 @@ spec:
       containers:
         - name: elasticsearch
           env:
-            - name: discovery.zen.ping.unicast.hosts
-              value: "elasticsearch-0.elasticsearch,elasticsearch-1.elasticsearch,elasticsearch-2.elasticsearch"
+            - name: cluster.initial_master_nodes
+              value: "elasticsearch-0,elasticsearch-1,elasticsearch-2"
+          readinessProbe:
+            $patch: replace
+            failureThreshold: 3
+            initialDelaySeconds: 45
+            periodSeconds: 10
+            successThreshold: 3
+            timeoutSeconds: 5
+            exec:
+              command:
+                - sh
+                - -c
+                - |
+                  #!/usr/bin/env bash -e
```

Comment:
🤯

Reply:
Copy pasted from elasticsearch helm chart. Tested along with @Reuenthal767 ! Nice team-work

Reply:
Yeah @lnovara that was my same reaction in the beginning, but what this readiness probe is doing is simply using a cheap trick (writing a file to /tmp) to differentiate between `cluster starting up` and `cluster already running` so that a suitable check can be performed.

I have used a similar readiness probe (just tailored for ES6) on 9 ES clusters for 2 weeks with very good results (especially regarding the nightly shutdown/restart of the underlying k8 cluster).

Also from what I've read in various github issues this readiness probe should help ensuring ES version upgrades are smooth.